src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -674,6 +678,36 @@ class FrontendGenImpl {
       mlir::emitWarning(UnknownLoc(),
           "Could not find op importer: assuming this "
           "represents a custom operator.");
+    mlir::emitWarning(UnknownLoc(), "Could not find op importer: assuming this "
+                                    "represents a custom operator.");
+    int nOps = node.input().size();
+    auto funcName = opName.str();
+    std::vector<mlir::Type> outputTypes;
+    std::vector<mlir::Value> inputs;
+    auto attributes = ImportNodeAttributes(node);
+    auto mlirAttr = builder_.getStringAttr(funcName);
+    auto funcAttr = builder_.getNamedAttr("function_name", mlirAttr);
```

Comment:
It would be useful to include the domain-name of the op also as an attribute.

Reply:
yep. done.